app/src/main/java/com/github/se/travelpouch/ui/profile/ProfileScreen.kt

```diff
@@ -103,54 +102,55 @@ fun ProfileScreen(navigationActions: NavigationActions, profileModelView: Profil
 
           LazyColumn(
               modifier = Modifier.fillMaxSize().padding(horizontal = 8.dp),
-              contentPadding = PaddingValues(bottom = 80.dp)) {
-                val friends = profile.value.friends.keys.toList()
+              // contentPadding = PaddingValues(bottom = 80.dp)
```

Comment:
Make sure not to leave any commented out lines like this

Reply:
This is done